js/calculatefunc.js

```diff
@@ -0,0 +1,99 @@
+const operate = require('./operate');
+
+const calculate = (input, calcObject) => {
+  let { num1, num2, displayValue, operator } = calcObject;
+
+  switch (input) {
+    case 'reset':
+      return { num1: null, num2: null, displayValue: null, operator: null };
```

Comment:
let's add test for this. Also, instead of `null` for `displayValue`, why not use and empty string `""`? This way it would be easy to display an empty string instead of null.